### DISCUSSION TITLE
Why I can't edit my figma UI Kit

### DISCUSSION THREAD

**Discussion by sia8933** [primefaces/community#3155] [Figma UI Kit] [answered]
Hi there,

I bought the Figma UI KIt for single designer and I got the license, how should I start using these ui kits? 
![QQ20241204-135307](https://github.com/user-attachments/assets/5ff50007-3d29-4772-9824-b57e7e1d6de2)

I saw that someone said to download the ui kit zip file inside the Primestore, but I don't see where I can download it.

I can see the kit in figma by clicking on it on this page, 
![QQ20241204-135945](https://github.com/user-attachments/assets/0b77df46-1ddf-4d0c-98cb-ad9ca866881d)

but it's not editable and says that I'm not the editor of the file, so what should I do?
![QQ20241204-140041](https://github.com/user-attachments/assets/e426bc0f-2765-4921-8456-a8bc75a9950c)

**Comment by sia8933** [comment]
Ok, I know where to download to ui kits.

**Comment by onursenture** [comment, marked as answer]
Hello, I'm glad you found the download. Let me also show the location for others who might encounter the same issue 👇

<img width="1539" alt="image" src="https://github.com/user-attachments/assets/87c3de22-6609-465a-9b8d-cd7c0330ef98" />
